### PR TITLE
Match with NLsolve new API

### PR DESCRIPTION
**Pull request by YingboMa**

### CODE REVIEW

File: REQUIRE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 julia 0.6
 DiffEqBase
 Reexport
-NLsolve
+NLsolve 0.12.1
```
Comment (ChrisRackauckas) [inline REQUIRE]: The argument flip is not until v0.14.0? @pkofod

Comment (pkofod) [reply REQUIRE]: Indeed

**Comment by ChrisRackauckas** [comment]
Let's wait on @pkofod to tag, then let CI pass and pull this in.

**Comment by pkofod** [comment]
I'm just waiting for metadata.

**Pull request merged**